src/Illuminate/Mail/Mailer.php

```diff
@@ -392,7 +390,11 @@ protected function sendSwiftMessage($message)
         }
 
         if (! $this->pretending) {
-            return $this->swift->send($message, $this->failedRecipients);
+            try {
+                return $this->swift->send($message, $this->failedRecipients);
+            } finally {
+                $this->swift->getTransport()->stop();
```

Comment:
Because calling "forceReconnection" would not make sense form a semantic point of view.